CoreFoundation/Base.subproj/SymbolAliases

```diff
@@ -1,105 +0,0 @@
-
-
-# This is specific to Swift open source; Foundation's NSCFConstantString is used as our constant string class reference
-__TMC15SwiftFoundation19_NSCFConstantString ___CFConstantStringClassReference
```

Comment:
Because they're also specified in DarwinSymbolAliases.
- The script used DarwinSymbolAliases for Darwin (Linux only has a single alias so it doesn't have a file)
- Xcode used SymbolAliases

It's better if they both use the same file. I decided to keep DarwinSymbolAliases because it's more obviously platform-specific.